demo/site/src/pageTypes/Page.tsx

```diff
@@ -37,6 +38,23 @@ export const pageQuery = gql`
     ${topMenuPageTreeNodeFragment}
 `;
 
+export async function loader({
+    client,
+    pageId,
+    contentScope,
+}: {
+    client: GraphQLClient;
+    pageId: string;
```

Comment:
Maybe rename this to `pageTreeNodeId` to avoid confusion between page tree node and page (= document)?

Reply:
done